tekton-catalog/pipeline-loops/pkg/reconciler/pipelinelooprun/pipelinelooprun_test.go

```diff
@@ -883,40 +847,6 @@ var expectedPipelineRunIteration1 = &v1beta1.PipelineRun{
 	},
 }
 
-var expectedConditionPipelineRunIteration1 = &v1beta1.PipelineRun{
```

Comment:
can you add some TODO comments for adding some condition pipeline unit tests? It's broken right now but we want to bring it back later on.

Reply:
Yes, I can. Alternatively, leave the issue open. If I get another opportunity, I will work on it.

Reply:
thanks

Reply:
Added a TODO. If you like, In the mean time, you can merge this PR